reference/3.0/Microsoft.PowerShell.Utility/ConvertFrom-Json.md

```diff
@@ -21,127 +21,128 @@ ConvertFrom-Json [-InputObject] <String> [<CommonParameters>]
 
 ## DESCRIPTION
 
-The **ConvertFrom-Json** cmdlet converts a JSON-formatted string to a custom object (PSCustomObject) that has a property for each field in the JSON string.
-JSON is commonly used by web sites to provide a textual representation of objects.
+The `ConvertFrom-Json` cmdlet converts a JavaScript Object Notation (JSON) formatted string to a
+custom **PSCustomObject** object that has a property for each field in the JSON string. JSON is
+commonly used by web sites to provide a textual representation of objects. The JSON standard does
+not prohibit usage that is prohibited with a **PSCustomObject**. For example, if the JSON string
+contains duplicate keys, only the last key is used by this cmdlet. See other examples below.
 
-To generate a JSON string from any object, use the **ConvertTo-Json** cmdlet.
+To generate a JSON string from any object, use the `ConvertTo-Json` cmdlet.
 
-This cmdlet is introduced in PowerShell 3.0.
+This cmdlet was introduced in PowerShell 3.0.
 
 ## EXAMPLES
 
-### Example 1
+### Example 1: Convert a DateTime object to a JSON object
 
+This command uses the `ConvertTo-Json` and `ConvertFrom-Json` cmdlets to convert a **DateTime**
+object from the `Get-Date` cmdlet to a JSON object.
+
+```powershell
+Get-Date | Select-Object -Property * | ConvertTo-Json | ConvertFrom-Json
 ```
-PS> Get-Date | Select-Object -Property * | ConvertTo-Json | ConvertFrom-Json
 
+```Output
 DisplayHint : 2
-
 DateTime    : Friday, January 13, 2012 8:06:31 PM
-
 Date        : 1/13/2012 8:00:00 AM
-
 Day         : 13
-
 DayOfWeek   : 5
-
 DayOfYear   : 13
-
 Hour        : 20
-
 Kind        : 2
-
 Millisecond : 400
-
 Minute      : 6
-
 Month       : 1
-
 Second      : 31
-
 Ticks       : 634620819914009002
-
 TimeOfDay   : @{Ticks=723914009002; Days=0; Hours=20; Milliseconds=400; Minutes=6; Seconds=31; TotalDays=0.83786343634490734; TotalHours=20.108722472277776; TotalMilliseconds=72391400.900200009; TotalMinutes=1206.5233483366667;TotalSeconds=72391.4009002}
-
 Year        : 2012
 ```
 
-This command uses the ConvertTo-Json and **ConvertFrom-Json** cmdlets to convert a **DateTime** object from the Get-Date cmdlet to a JSON object.
-
-The command uses the Select-Object cmdlet to get all of the properties of the **DateTime** object.
-It uses the **ConvertTo-Json** cmdlet to convert the **DateTime** object to a JSON-formatted string and the **ConvertFrom-Json** cmdlet to convert the JSON-formatted string to a JSON object..
+The example uses the `Select-Object` cmdlet to get all of the properties of the **DateTime** object.
+It uses the `ConvertTo-Json` cmdlet to convert the **DateTime** object to a JSON-formatted string
+and the `ConvertFrom-Json` cmdlet to convert the JSON-formatted string to a JSON object.
```

Comment:
Is it a JSON object or a PSCustomObject?  Same for all versions.

Reply:
Fix the object type confusion.